README.md

```diff
@@ -12,7 +12,7 @@ git clone https://github.com/Smeks-ops/maca-assessment.git
 
 ## Installing the App
 
-2. Install dependencies with this command in the home folder, the api-server folder and the client folder
+2. Install dependencies with this command in the home folder, the api-server folder and the client folder 
```

Comment:
Nitpick: installation was a bit confusing here to have to run the same command in multiple directories

Reply:
Ok yeah because I built it as a mono repo, I had three package.jsons